### ISSUE TITLE
Test Linting guides for Python/JS

### ISSUE THREAD

**Issue by BenMatase** [BucknellSustainability/Flower#154]

**Comment by BenMatase** [comment]
This can't be done until the PR for both get merged in.  I'll comment when that happens

**Comment by Phlosioneer** [comment]
@BenMatase This is why Dependencies are a thing.

**Comment by BenMatase** [comment]
Oh true, I had a brain fart. I thought there weren't open issues related but there are. I'll fix that rn

**Comment by Phlosioneer** [comment]
Yeah, the "Done" label is misleading. Perhaps we should rename that to "Awaiting Review" or somesuch.

**Comment by BenMatase** [comment]
That is good to know.  We can talk about this tomorrow
